bc_obps/pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[tool.poetry]
+name = "reporting"
+version = "0.1.0"
+description = ""
+authors = ["Josh Gamache <joshua@button.is>"]
```

Comment:
probably `ggircs@gov.bc.ca` here - unless you want to be on the hook as the point of contact :p

Reply:
I would rather not have that responsibility 😆. Changed in b19a3c1.